homeassistant/components/comelit/sensor.py

```diff
@@ -0,0 +1,83 @@
+"""Support for sensors."""
+from __future__ import annotations
+
+from typing import Final
+
+from aiocomelit import ComelitSerialBridgeObject
+from aiocomelit.const import OTHER
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import UnitOfPower
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import ComelitSerialBridge
+
+SENSOR_TYPES: Final = (
+    SensorEntityDescription(
+        key="power",
+        translation_key="power",
```

Comment:
```suggestion
```
Can use device class translations

Reply:
I just saw this approach used by other integrations and applied it here ;-)